lib/sync/fetcher.go

```diff
@@ -84,7 +86,7 @@ func (f *BlockFetcher) Fetch(ctx context.Context, syncInfo *SyncInfo) (*SyncInfo
 					return false, ctx.Err()
 				}
 
-				f.logger.Error(err.Error(), "err", err)
+				f.logger.Error(fmt.Sprintf("fetch err: %v", err), "err", err, "height", height)
```

Comment:
`"error"` already exists, so instead `fmt.Sprintf("fetch err: %v", err),` `"fetch err"`will be enough.

Reply:
Ok,